src/util/uint.rs

```diff
@@ -516,13 +516,13 @@ impl Uint256 {
     #[inline]
     pub fn increment(&mut self) {
         let &mut Uint256(ref mut arr) = self;
-        arr[0] += 1;
+        arr[0] = arr[0].wrapping_add(1);
         if arr[0] == 0 {
-            arr[1] += 1;
+            arr[1] = arr[1].wrapping_add(1);
             if arr[1] == 0 {
-                arr[2] += 1;
+                arr[2] = arr[2].wrapping_add(1);
                 if arr[2] == 0 {
-                    arr[3] += 1;
+                    arr[3] = arr[3].wrapping_add(1);
```

Comment:
Shouldn't we let the last one panic?

Reply:
In `Uint256`, add and multiply operation also don't panic when overflow. I think increment should follow their behavior, so if we prefer panic, we should change them too.